app/exception/envelope_handlers.py

```
@@ -92,38 +92,51 @@ async def global_exception_handler_envelope(request: Request, exc: Exception):
     모든 예외(500 포함)를 ApiResponse 포맷으로 변환
     
     Rationale:
-        예상치 못한 서버 에러가 발생해도 프론트엔드는 항상
-        isSuccess: false 형태의 JSON을 받게 됩니다.
+        예상치 못한 서버 에러 발생 시 상세 스택 트레이스는 로그에만 기록하고,
+        클라이언트에게는 일반적인 메시지만 반환하여 보안을 강화합니다.
     """
+    # 상세 로그 기록 (Trace ID는 로깅 필터에서 자동으로 주입됨)
     logger.exception(
-        "Unhandled Exception",
+        f"Unhandled Exception: {str(exc)}",
```

Ruff (0.15.0)

[warning] Use explicit conversion flag

Replace with conversion flag

(RUF010)